pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -466,6 +466,33 @@ public static KitRequestShipping getKitRequestShipping(@NonNull ResultSet rs) th
         return kitRequestShipping;
     }
 
+    public static KitRequestShipping getKitRequestShippingForResample(@NonNull ResultSet rs) throws SQLException {
```

Comment:
The kit for resample does not need some information that comes from other non-kit related table like kit display name and participant status, ups tracking, etc... So this method only reads what is in the two kit tables (ddp_kit_request and ddp_kit) tables and creates a kit from them

Reply:
Why not use an existing method that returns KitRequestShipping?  Since this is an admin function, there's probably not a performance issue if existing ways to get KitRequestShipping hit extra tables that aren't needed for repatienting.  The only thing that would be needed is to get from collaboratorSampleId to whatever id is needed to pass into an existing method that returns a KitRequestShipping object.